app/controllers/api/base_controller/providers.rb

```diff
@@ -1,14 +1,14 @@
 module Api
   class BaseController
     module Providers
-      TYPE_ATTR         = "type"
-      ZONE_ATTR         = "zone"
-      CREDENTIALS_ATTR  = "credentials"
-      AUTH_TYPE_ATTR    = "auth_type"
-      DEFAULT_AUTH_TYPE = "default"
+      TYPE_ATTR         = "type".freeze
+      ZONE_ATTR         = "zone".freeze
+      CREDENTIALS_ATTR  = "credentials".freeze
+      AUTH_TYPE_ATTR    = "auth_type".freeze
+      DEFAULT_AUTH_TYPE = "default".freeze
       CONNECTION_ATTRS  = %w(connection_configurations).freeze
       ENDPOINT_ATTRS    = %w(hostname ipaddress port security_protocol).freeze
-      RESTRICTED_ATTRS  = [TYPE_ATTR, CREDENTIALS_ATTR, ZONE_ATTR, "zone_id"]
+      RESTRICTED_ATTRS  = [TYPE_ATTR, CREDENTIALS_ATTR, ZONE_ATTR, "zone_id"].freeze
```

Comment:
Please don't mix unrelated rubocops in the same PR and the feature itself (i.e. these freezes).